protocol/protocol.go

```diff
@@ -642,8 +642,6 @@ func (o *Operation) UnmarshalBinary(body []byte) error {
 			// ignore padding
 		case TagCustomFuncName:
 			o.CustomFuncName = string(data)
-		default:
-			return fmt.Errorf("unknown tag: %02x", tag)
```

Comment:
I think you want to continue here instead. See the comment on the next line.

Reply:
Ah I think you're right, I was looking at the comment for `TagPadding` but I guess that's different because it's both known and ignored.